src/client/camera.cpp

```diff
@@ -573,12 +573,12 @@ void Camera::updateViewingRange()
 
 	m_cameranode->setNearValue(0.1f * BS);
 
-	m_draw_control.wanted_range = std::fmin(adjustDist(viewing_range, getFovMax()), 4000);
+	m_draw_control.wanted_range = std::fmin(adjustDist(viewing_range, getFovMax()), 6000);
 	if (m_draw_control.range_all) {
 		m_cameranode->setFarValue(100000.0);
 		return;
 	}
-	m_cameranode->setFarValue((viewing_range < 2000) ? 2000 * BS : viewing_range * BS);
+	m_cameranode->setFarValue((m_draw_control.wanted_range < 2000) ? 2000 * BS : m_draw_control.wanted_range * BS);
```

Comment:
```suggestion
	m_cameranode->setFarValue(std::fmax(m_draw_control.wanted_range, 2000) * BS);
```

Reply:
Sure. I even wrote it like the description :)